swig/swigResqml2_0_1Include.i

```diff
@@ -140,7 +140,6 @@ namespace std {
    %template(StratigraphicUnitInterpretationVector) vector<RESQML2_0_1_NS::StratigraphicUnitInterpretation*>;
    %template(StratigraphicOccurrenceInterpretationVector) vector<RESQML2_0_1_NS::StratigraphicOccurrenceInterpretation*>;
    %template(HorizonInterpretationVector) vector<RESQML2_0_1_NS::HorizonInterpretation*>;
-   %template(ConstGrid2dRepresentationVector) vector<RESQML2_0_1_NS::Grid2dRepresentation const*>;
```

Comment:
Should it not be replaced by the non const version?

Reply:
Sure. Done.

Reply:
Sure. Done.